tls/test/test_hello_message.py

```diff
@@ -139,6 +139,23 @@ class TestClientHello(object):
         b'\x00\x04'
     ) + truncated_hmac_ext_packet
 
+    cert_status_request_ext = (
```

Comment:
> I need to come up with some test bytes that's not zero-length request_extensions and responder_id_list.

I think it's important to have these tests, but I also think we're reaching a point where we can parse more extensions, and more instantiations of extensions, than are in common use.  I don't want to prevent this PR from landing because of how difficult it is to get actual implementations to utter a status request message.

More generally I think it's OK for now to test a single instantiation of an extension and then use #104 to drive a more comprehensive test suite.